### ISSUE TITLE
 error CS8034

### ISSUE THREAD

**Issue by ErikEJ** [thepirat000/Audit.NET#297]
**Describe the bug**
'dotnet test' fails with this error, when building with Audit.Net included.

Removing any reference to audit.net causes the build to pass.

Error:

CSC : error CS8034: Unable to load Analyzer assembly C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.aspnetcore.mvc.analyzers\2.2.0\analyzers\dotnet\cs\Microsoft.AspNetCore.Mvc.Analyzers.dll : Assembly with same name is already loaded 

``` xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
  </PropertyGroup>
  
  <PropertyGroup>
   <RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Audit.WebApi.Core" Version="15.0.4" />
    <PackageReference Include="Audit.NET.AzureStorage" Version="15.0.4" />
    <PackageReference Include="Towel" Version="1.0.11-alpha" />
  </ItemGroup>

  </ItemGroup>

</Project>
```

dotnet version: 3.1.201

**Additional context**
related to #275 

Also see https://github.com/dotnet/websdk/issues/1203 Quote: "The Microsoft.AspNetCore.Mvc 2.2.0 package should not be used with 3.0 or later targeting projects. "

**Comment by ErikEJ** [comment]
So I am kindly asking for a ASP.NET Core 3.1 release - Let me know if I can help with a PR?

**Comment by thepirat000** [comment]
Did you tested also with latest Audit.NET version?

I think this was fixed on commit https://github.com/thepirat000/Audit.NET/commit/713ca904064b2939a50a0ab1a2ebc5f9572f08e7

**Comment by ErikEJ** [comment]
Yes, I have tested 15.2.1 - it still depends on Micorsoft.AspNetCore.Mvc 2.2, so no difference

**Comment by thepirat000** [comment]
PR is very welcome

**Comment by thepirat000** [comment]
This is fixed on version 15.2.2.

Thanks

**Comment by ErikEJ** [comment]
Thanks - works like a charm now!

and I could remove this:

   <PropertyGroup>
     <RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
  </PropertyGroup>
